app/jobs/descriptive_metadata_import_job.rb

```diff
@@ -23,6 +23,7 @@ def perform(bulk_action_id, params)
                        .bind { |description| validate_changed(cocina_object, description) }
                        .bind { |description| open_version(cocina_object, description) }
                        .bind { |description, new_cocina_object| CocinaValidator.validate_and_save(new_cocina_object, description:) }
+                       .bind { |new_cocina_object| close_version(new_cocina_object) }
                        .either(
```

Comment:
`new_cocina_object` does not appear to be getting returned from the `CocinaValidator.validate_and_save(new_cocina_object, description:)` call despite the fact that https://github.com/sul-dlss/argo/blob/main/app/services/cocina_validator.rb#L16 calls https://github.com/sul-dlss/argo/blob/main/app/models/repository.rb#L14-L17, which does appear to return the object